pingcap/tidb
# brie: integrate lightning to suport IMPORT statement

### What problem does this PR solve?

Problem Summary:

### What is changed and how it works?

What's Changed:

implement `IMPORT` statement by integrating lightning

```
mysql> import database * from '/tmp/test';
+---------+--------+-------------+------+---------------------+---------------------+---------------------+---------+
| Task ID | Status | Data Source | Size | Queue Time          | Execution Time      | Finish Time         | Message |
+---------+--------+-------------+------+---------------------+---------------------+---------------------+---------+
|       1 | Finish | /tmp/test   |    0 | 2020-11-23 10:58:56 | 2020-11-23 10:58:56 | 2020-11-23 10:59:00 |         |
+---------+--------+-------------+------+---------------------+---------------------+---------------------+---------+
1 row in set (4.45 sec)
```

How it Works:

construct a configuration for lightning, use `glue` interface to communicate, create `mysql.brie_tasks` at bootstrap to save metadata of IMPORT task

For this PR, resuming and remove history job or failed job are not included

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
  use IMPORT statement with correct/wrong local disk datasource. check statement result, SHOW IMPORTS, SHOW PROCESSLIST

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM

### Release note 

- No release note
